src/01-dinosaur-facts.js

```diff
@@ -71,7 +97,22 @@ function getDinosaurDescription(dinosaurs, id) {}
  *  getDinosaursAliveMya(dinosaurs, 65, "unknown-key");
  *  //> ["WHQcpcOj0G"]
  */
-function getDinosaursAliveMya(dinosaurs, mya, key) {}
+const getDinosaursAliveMya = (dinosaurs, mya, key) => {
+  return dinosaurs.filter(dino => {
+    const myaArr = dino.mya;
+    if (myaArr.length === 1) {
+      return [myaArr[0], myaArr[0] - 1, myaArr[0] + 1].includes(mya);
```

Comment:
this is clever - but its a little hard code-y for my liking, also the logic can be a bit confusing.  

Can we refactor this to make reference the data more directly rather than adding or subtracting one from the value at `.mya[0]` ?